jdaviz/configs/mosviz/helper.py

```diff
@@ -55,6 +56,10 @@ def __init__(self, *args, **kwargs):
         self.app.hub.subscribe(self, TableClickMessage,
                                handler=self._row_click_message_handler)
 
+        # Listen for new redshifts from the redshift slider (NOT YET IMPLEMENTED)
```

Comment:
Does this mean this PR is incomplete? Why listen to an unimplemented event?

Reply:
the slider itself isn't implemented in the UI yet (in progress in an upcoming PR), but this currently catches messages from the API equivalent (`mosviz.specviz.set_redshift`).